modules/fts/pages/fts-query-types.adoc

```diff
@@ -379,6 +384,7 @@ It performs an exact term-match for all the phrase-constituents, without using a
 
 A phrase query is also demonstrated by means of the Java SDK, in xref:2.7@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
 
+
```

Comment:
Unnecessary new line. Also need to mention that fuzzy queries are non-analytic too.

Reply:
done!